common.mk

```diff
@@ -74,6 +74,10 @@ check_aws: check_python
 		false; \
 	fi
 
+.PHONY: rename_resources
+rename_resources:
+	python $(project_root)/scripts/rename_resources.py
```

Comment:
I'm assuming that you've checked this with Hannes, but I'm a little surprised that we want a make target for this script. My understanding was that this was something that would only need to be run once for each deployment. Then the script could be archived in a later PR.

Reply:
Yeah, Hannes requested that the script be injected into the build process specifically. I'm not sure if there's a strong advantage beyond keeping UPGRADING.rst a little smaller. I'll raise this during standup.

Reply:
It's ok, I was just surprised. If that's what you were instructed to do, it sounds good to me :+1:

Reply:
@hannes-ucsc

Reply:
The size of UPGRADING.rst isn't a concern but the need for manual upgrade intervention is. Implementing this as specified in the ticket means avoiding the manual intervention when these changes are deployed to any deployment. I'm happy to discuss this further, but be sure to read the ticket description first, @jessebrennan.